ui/app/services/config.js

```diff
@@ -0,0 +1,9 @@
+import Service from '@ember/service';
+
+export default Service.extend({
```

Comment:
With octane you should be able to start writing these as Native classes (however, ignore me if we have to backport to prior to 1.7). 
ex: 
```
import Service from '@ember/service';

export default class ShoppingCartService extends Service {
}
```

Reply:
Great catch, however we do indeed have to backport this to prior to 1.7 😭